### PR TITLE
52 - About section our values

### PR DESCRIPTION
**Pull request by terrianx**
Values component for about section. 

- Not desktop responsive, at screen width 1175px 100% zoom view text wraps around and is misaligned. 
- Also not mobile responsive, but Dominic mentioned he would take care of it. 
- Shares CSS modules with belief CSS module and includes its own for formatting SaLT CoMManD and also requiring a specific ordering the Collapsible components.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "chalk": "^5.3.0",
     "cli-spinner": "^0.2.10",
-    "shelljs": "^0.8.5"
+    "shelljs": "^0.8.5",
+    "tailwindcss": "^3.3.3"
```
Comment (thedomcab) [inline package.json:13]: Why did you add tailwindcss in root? it is only needed in frontend.

Comment (terrianx) [reply package.json:13]: my bad, i was in wrong directory when i installed it. ill push again with these lines deleted.

File: frontend/src/components/page-about/index/values.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    <div className={titleContainer}>
+      <div className="subheading">Our Values</div>
+      <div className={title}>
+        <h2>S</h2>
+        <h3>a</h3>
+        <h2>LT C</h2>
+        <h3>o</h3>
+        <h2>MM</h2>
+        <h3>an</h3>
+        <h2>D</h2>
```
Comment (thedomcab) [inline frontend/src/components/page-about/index/values.js:64]: this is not proper way to use H# tags.  you use one only the whole heading and use span with different classes to give the effect we want. Could you rework this? This will definitely hurt our SEO (Search Enging Optimization).  Google will definitely not like this when they crawl our website.  Could you rethink a better way to do this? please

Comment (terrianx) [reply frontend/src/components/page-about/index/values.js]: you are right, ive pushed up a change to address this.

**Review by thedomcab — APPROVED** [review]
